src/Pact/Types/Term.hs

```diff
@@ -847,3 +1063,12 @@ makeLenses ''App
 makeLenses ''Def
 makeLenses ''ModuleName
 makePrisms ''DefType
+
+deriveEq1 ''App
```

Comment:
Did `deriveEq1` for `Term` not work (because Bound presumably)?

Reply:
that's exactly right :disappointed:

Reply:
Boo